core/pta/bcm_pta/sub.mk

```diff
@@ -0,0 +1,2 @@
+global-incdirs-y += include
```

Comment:
I don't suppose any C files will use `pta_bcm_*.h` files but files in/under  this directly.
How about putting everything directly in this directory with no further hierarchy (skipping the separate `src` directory too)?

Reply:
Agreed.